src/main/java/com/serotonin/bacnet4j/obj/TrendLogMultipleObject.java

```diff
@@ -175,8 +176,10 @@ public boolean isLogDisabled() {
         return logDisabled;
     }
 
-    public LogBuffer<LogMultipleRecord> getBuffer() {
-        return buffer;
+    public void withBuffer(Consumer<LogBuffer<LogMultipleRecord>> bufferConsumer) {

```

Comment:
This is one of the 2 changes that make the tests pass, accessing the buffer in a thread safe manner.